### PR TITLE
Adding forta alert sdk docs

### PR DESCRIPTION
**Pull request by Robert-H-Leonard**

### CODE REVIEW

File: docs/python.md [hunk 1]
```diff
@@ -154,6 +154,34 @@ If a bot wants to flag a transaction/block because it meets some condition (e.g.
...
+- `protocol` - name of protocol being reported on
+- `scanNodeCount` - number of scanners that found the alert
+- `source` - source where the alert was detected
+- `projects` - list of Web3 projects related to the alert
+- `findingType` -  indicates type of finding:
+    - Exploit
+    - Suspicious
+    - Degraded
+    - Info
+    - Umnknown_Type
```
Comment (haseebrabbani) [inline docs/python.md:176]: Unknown*

File: docs/sdk.md [hunk 2]
```diff
@@ -158,6 +158,34 @@ If a bot wants to flag a transaction/block because it meets some condition (e.g.
...
+- `addresses` -  human-readable list of addresses involved in the alert
+- `alertId` -  unique string to identify this class of finding
+- `contracts` -  list of contracts related to the alert
+- `createdAt` -  timestamp when the alert was published
+- `description` - text description of the alert
+- `name` - alert name
+- `protocol` - name of protocol being reported on
+- `scanNodeCount` - number of scanners that found the alert
+- `source` - source where the alert was detected
+- `projects` - list of Web3 projects related to the alert
```
Comment (haseebrabbani) [inline docs/sdk.md:174]: lets elaborate on the fields these have

File: docs/sdk.md [hunk 3]
```diff
@@ -158,6 +158,34 @@ If a bot wants to flag a transaction/block because it meets some condition (e.g.
...
+- `findingType` -  indicates type of finding:
+    - Exploit
+    - Suspicious
+    - Degraded
+    - Info
+    - Umnknown_Type
```
Comment (haseebrabbani) [inline docs/sdk.md:180]: Unknown*

File: docs/sdk.md [hunk 4]
```diff
@@ -170,6 +198,39 @@ A convenience function called `getJsonRpcUrl` can be used to load a JSON-RPC URL
...
+- `scanNodeConfirmations` - filter alerts by number of scan nodes confirming the alert
+- `severities` - filter alerts by severity levels
+- `transactionHash` - indicate a transaction hash, alerts returned will only be from that transaction
+- `blockSortDirection` - indicate sorting order by block number, 'desc' or 'asc'. Default is 'desc'.
+- `blockDateRange` - alerts returned will be between the specified start and end block timestamp dates when the threats were detected
+- `blockNumberRange` - alerts for the block number range will be returned
+
+The returned alerts are formatted to match the SDK `AlertsResponse` interface the looks like:
+
+```javascript
```
Comment (haseebrabbani) [inline docs/sdk.md:222]: a short usage example would help here as well

File: docs/python.md [hunk 5]
```diff
@@ -154,6 +154,34 @@ If a bot wants to flag a transaction/block because it meets some condition (e.g.
...
+- `severity` - indicates impact level of finding:
+    - Critical - exploitable vulnerabilities, massive impact on users/funds
+    - High - exploitable under more specific conditions, significant impact on users/funds
+    - Medium - notable unexpected behaviours, moderate to low impact on users/funds
+    - Low - minor oversights, negligible impact on users/funds
+    - Info - miscellaneous behaviours worth describing
+- `metadata` - key-value map (both keys and values as strings) for providing extra information
```
[the remark this answers is not in the document]
Comment (Robert-H-Leonard) [reply docs/python.md]: Updated to snake-case

File: docs/python.md [hunk 6]
```diff
@@ -166,6 +194,27 @@ A convenience function called `get_json_rpc_url` can be used to load a JSON-RPC
...
+- `createdSince` - indicate number of milliseconds, alerts returned will be alerts created since the number of milliseconds indicated ago
+- `first` - indicate max number of results.
+- `startingCursor` - query results after the specified cursor
+- `projectId` - indicate a project id, alerts returned will only be from that project.
+- `scanNodeConfirmations` - filter alerts by number of scan nodes confirming the alert
+- `severities` - filter alerts by severity levels
+- `transactionHash` - indicate a transaction hash, alerts returned will only be from that transaction
+- `blockSortDirection` - indicate sorting order by block number, 'desc' or 'asc'. Default is 'desc'.
+- `blockDateRange` - alerts returned will be between the specified start and end block timestamp dates when the threats were detected
+- `blockNumberRange` - alerts for the block number range will be returned
```
[the remark this answers is not in the document]
Comment (Robert-H-Leonard) [reply docs/python.md]: Updated

File: docs/sdk.md (hunk 4 above)
Comment (Robert-H-Leonard) [reply docs/sdk.md:231]: Added a usage example.

**Review by haseebrabbani — APPROVED** [review]

**Pull request merged**